Riot/Modules/Room/RoomViewController.m

```diff
@@ -243,7 +243,9 @@ @interface RoomViewController () <UISearchBarDelegate, UIGestureRecognizerDelega
 @property (nonatomic, strong) RoomMessageURLParser *roomMessageURLParser;
 @property (nonatomic, strong) RoomCreationModalCoordinatorBridgePresenter *roomCreationModalCoordinatorBridgePresenter;
 @property (nonatomic, strong) RoomInfoCoordinatorBridgePresenter *roomInfoCoordinatorBridgePresenter;
+@property (nonatomic, strong) RoomCoordinatorBridgePresenter *threadBridgePresenter;
 @property (nonatomic, strong) CustomSizedPresentationController *customSizedPresentationController;
+@property (nonatomic, strong) ThreadsCoordinatorBridgePresenter *threadsCoordinatorBridgePresenter;
```

Comment:
This seems fairly confusing having a `threadBridgePresenter` to display a single thread, and a `threadsCoordinatorBridgePresenter` to display the threads list which can also display another thread on top. It would make sense to me to have one of the following:
- Rename the latter to `threadsListBridgePresenter` (Which I realise is technically not accurate but is more descriptive about its purpose).

or
- A single `threadsCoordinatorBridgePresenter` that can be configured to show a single thread *or* the threads list (this seems the most obvious but I appreciate is a *lot* more work).

Reply:
Totally agree. Actually `threadBridgePresenter` was an old thing, thanks for reminding me to refactor this part :)

Reply:
Much nicer 🎉